hack/lib/golang.sh

```diff
@@ -558,6 +560,17 @@ kube::golang::build_binaries() {
     local use_go_build
     local -a targets=()
     local arg
+    
+    # Add any files with those //generate annotations in the array below.
+    readonly BINDATAS=("./test/e2e/framework/gobindata_util.go" )
```

Comment:
Isn't there a path mundger?  This is relative from execution so if you're doing things out of tree it will fail.